cmd/kubeadm/app/cmd/init.go

```diff
@@ -21,7 +21,9 @@ import (
 	"io"
 	"io/ioutil"
 	"os"
+	"os/user"
```

Comment:
Will this impact kubeadm's ability to be statically linked? Does this work on Windows as well?
In some cases I've seen very strange things happen when using this package in Go...

Reply:
I worry that this might break our "statically linked" guarantee of the kubeadm program or mess something up with the flags needed to be passed to the `go build` command... I've been here before _sigh_ https://github.com/kubernetes/kubernetes/pull/26028

Reply:
i guess the whole point of me using the package was portability - otherwise i would have used `bash` and `os/exec`. so according to https://golang.org/pkg/os/user/ i need to add some runtime `if` to see what the OS is and behave slightly differently.

Reply:
i cannot answer the `static link` question. maybe someone else can in terms of this particular package.